src/EnergyPlus/CurveManager.cc

```diff
@@ -2997,7 +2997,7 @@ namespace Curve {
 
     bool CheckCurveDims(EnergyPlusData &state,
                         int const CurveIndex,
-                        std::vector<int> validDims,
+                        std::vector<int> const &validDims,
```

Comment:
Come on people, you're better than this.

Reply:
This is a focus group